site/profile/manifests/cvmfs.pp

```diff
@@ -48,6 +49,15 @@
     require => Package['cvmfs']
   }
 
+  file { '/etc/cvmfs/config.d.ctmpl':
```

Comment:
Shouldn't `/etc/cvmfs/config.d` be a directory?

Reply:
Done